src/models/User.model.js

```diff
@@ -79,5 +82,13 @@ export default (sequelize, DataTypes) => {
     }
   });
 
+  const withoutPassword = async (user) => {
+    delete user.dataValues.password;
```

Comment:
tạo object mới mà k có password thì an toàn hơn á. Cái ni mi đang sửa object đầu vào nhiều khi sai á nhe